### DISCUSSION TITLE
What is your favorite movie

### DISCUSSION THREAD

**Discussion by KantaMatsumoto** [NAIST-SE/AttractingContributors#182] [General]
Hi, My name is Kanta. Please tell me about your favorite movie.
This is my favorite movies.
https://github.com/KantaMatsumoto/AttractingContributors/blob/main/README.md

**Comment by YPanY** [comment]
added Batman. pls help me by adding your favorite programming language or framework here👇 THX!
https://github.com/Pan-NLP-NAIST/AttractingContributors-programming_language/blob/main/README.md

**Comment by tktkbohshi** [comment]
Added!
please help me!
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Comment by t-matsumoto4758** [comment]
Added!
please check and help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by TakumiHiraide** [comment]
Added!! 
Please check this link. Help me ~~
https://github.com/TakumiHiraide/My-favorite-country

**Comment by pgDora56** [comment]
I sent a PR, please check it.

I need your help! Tell me your favorite youtuber!
https://github.com/pgDora56/AttractingContributors-Youtuber

**Comment by TakumiHiraide** [comment]
Added !! Please check me!!
https://github.com/TakumiHiraide/Where-do-you-want-to-go-

**Comment by ellielo** [comment]
I have few of them:
- Forrest Gump
- Green book
- La-la-lend
- Gifted
Sometimes I just like to watch tv shows on peacock tv [https://www.firesticktricks.com/peacock-tv-apk-firestick.html](url). Discovered it for myself recently but must admit they have some great stuff available.
